Gemfile

```diff
@@ -4,33 +4,31 @@ eval_gemfile(File.expand_path("gems/pending/Gemfile", __dir__))
 # VMDB specific gems
 #
 
+gem "activerecord-deprecated_finders", "~>1.0.4",  :require => "active_record/deprecated_finders"
 gem "rails",                           RAILS_VERSION
-gem "activerecord-deprecated_finders", "~>1.0.4",     :require => "active_record/deprecated_finders"
 
 # Client-side dependencies
-gem "jquery-rjs", "=0.1.1", :git => 'https://github.com/amatsuda/jquery-rjs.git'
-gem 'angular-ui-bootstrap-rails', '~> 0.13.0'
-gem 'momentjs-rails', '~> 2.10.3'
-gem 'jquery-rails', "~>4.0.4"
-gem 'jquery-hotkeys-rails'
-gem 'codemirror-rails', "=4.2"
-gem 'lodash-rails', '~> 3.10.0'
-
-# On MS Windows run "bundle config --local build.libv8 --with-system-v8" first
-
-gem 'sass-rails'
-gem 'patternfly-sass', "~>1.3.1"
-gem 'bootstrap-datepicker-rails'
+gem "angular-ui-bootstrap-rails",     "~>0.13.0"
+gem "angularjs-rails",                "~>1.4.3"
```

Comment:
Bad rebase, after #3940 (which is already merged), there should be no `angularjs-rails` anywhere.